### DISCUSSION TITLE
HELP IN TTL BONDING AND CONFIGURATION

### DISCUSSION THREAD

**Discussion by beytullh38** [arkypita/LaserGRBL#1369] [Q&A]
I HAVE DONE A PROJECT LASER ENGRAVING MACHINE BUT I COULD NOT FUCK THE CONFIGURATION AND TTL CONNECTION PART. PLEASE WAIT YOUR HELP.
THANK YOU
![1](https://user-images.githubusercontent.com/80915495/111671436-fe838180-8829-11eb-8a1a-81d7f4b18936.PNG)
![2](https://user-images.githubusercontent.com/80915495/111671443-ffb4ae80-8829-11eb-8be7-0fdadb29b195.PNG)

HOW CAN I CONNECT THIS 5.5 W LASER TO TTL

![aaef1973-77b7-4f4f-a6d6-9dc338372bb2](https://user-images.githubusercontent.com/80915495/111671924-7782d900-882a-11eb-8fde-3aa573f7c696.jpg)

 THIS IS THE LASER SKELETON SIZE

nema 17 17hs4401 THE STEP ENGINE I USE   200 STEPS

I ARE WAITING FOR YOUR HELP, THANKS SO MUCH NOW

**Comment by StuartB4** [comment]
Which control board do you have? Need more information.

**Reply by beytullh38** [reply]
Arduino uno and cnc shild

**Reply by beytullh38** [reply]
![Uploading Screenshot_2021-03-20-17-14-45-93_7a4090f09f6554852d748ee9fd6f40d3.png…]()

**Comment by StuartB4** [comment]
Like this.

![image](https://user-images.githubusercontent.com/16980026/111873279-76b38980-8987-11eb-8c65-f2f6f90e2648.png)

**Comment by StuartB4** [comment]
If your CNC shield is version 3 and you will use GRBL 1.1 connect it as above in the picture.

If it is a version 3.51 shield connect the TTL+ to the SpnEN pin and the TTL- to the Ground pin next to it on the shield.
